Doc/whatsnew/3.8.rst

```diff
@@ -148,6 +148,20 @@ extensions compiled in release mode and for C extensions compiled with the
 stable ABI.
 (Contributed by Victor Stinner in :issue:`36722`.)
 
+f-strings now support !d for quick and dirty  debugging
+-------------------------------------------------------
+
+Add ``!d`` conversion specifier to f-strings. ``f'{expr!d}'`` expands
```

Comment:
Since this is an alternative to PR that implements `!d` I am just adding a note to update this in case this PR is merged to use `=` in 3.8 .